### PR TITLE
feat: info-bites

### PR DESCRIPTION
**Pull request by aybarsayan**
## feat: Info Bites Tamplate KILTProtocol/ticket#3205
Please include a summary of the changes provided with this pull request and which issue has been fixed. 
Please also provide some context if necessary.

## How to test:
Please provide a brief step-by-step instruction.
If necessary provide information about dependencies (specific configuration, branches, database dumps, etc.)

- Step 1
- Step 2
- etc.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [ ] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

File: info-bites/july-2023.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# July Info Bites
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:1]: @aybarsayan Is the name and file name correct? July 2023? Though I guess it doesn't matter

File: info-bites/july-2023.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## 1
+
+![what is DID](files/whatisdid.jpeg)
+
+🍪 Information Bites 🍪
+
+What is DIDs?
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:9]:
```suggestion
What are DIDs? 
```

File: info-bites/july-2023.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## 1
+
+![what is DID](files/whatisdid.jpeg)
+
+🍪 Information Bites 🍪
+
+What is DIDs? 
+
+- In our current system, we have to rely on centralized authority (such as government) or singular monopoly services (Google, Facebook, etc.) for our digital identity. They are storing our data in centralized databases and control our data.
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:11]:
```suggestion
- Currently, we have to rely on a centralized authority (such as a government) or singular monopoly (Google, Facebook, etc.) for our digital identity. They store our data in centralized databases and control our data. 
```

File: info-bites/july-2023.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+- A KILT decentralized identifier (DID) is like a magical code that uniquely identifies each KILT user. It's like their secret key to the digital realm! 🧙‍♂️✨
+
+For example, did:kilt:4sxSYXakw1ZXBymzT9t3Yw91mUaqKST5bFUEjGEpvkTuckar. Impressive, isn't it? 😮💫
+
+- And here's the best part: DIDs can be used for people, machines, services, and anything else that wants to have an identity. It's a party where everyone's invited! 🎉🤖💼
+
+- Now, let's talk about the two types of DIDs: Light DIDs and Full DIDs. Light DIDs are like cool secret agents working offline, while Full DIDs are the superheroes residing on the chain, ready to save the day! 🕵️‍♂️🦸‍♀️🔒
+
+- But wait, there's more! A Full DID comes with some special powers. It supports an attestation key, which is used to write CTypes and attestations on the blockchain. It's like leaving your mark in the digital world! 📝🔏
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:39]: I get the impression that this is for beginners, will they have any idea what a CType or an attestation is?

File: info-bites/july-2023.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+- And that's not all! A Full DID also has a delegation key, which is used to write delegations on the blockchain.
+
+So, with KILT DIDs, get ready to unlock your identity and embark on exciting journeys! 🗝️ Claim Independence! 🙌✨
+
+## 3
+
+![light vs full](files/deloitte.jpeg)
+
+🧐 Who is? 🧐
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:49]: Do you mean to have the "Delloite" part here?

File: info-bites/july-2023.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+Who is Delloite?
+
+- Deloitte offers integrated services that include Audit & Assurance, Consulting, Financial Advisory, Risk Advisory and Tax & Legal. Our approach combines insight and innovation from multiple disciplines with business and industry knowledge to help our clients excel anywhere in the world.
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:53]:
```suggestion
- Deloitte offers integrated services that include Audit & Assurance, Consulting, Financial Advisory, Risk Advisory and Tax & Legal. Their approach combines insight and innovation from multiple disciplines with business and industry knowledge to help clients excel anywhere in the world.
```

Unless this is supposed to be said by someone from Delloite?

File: info-bites/july-2023.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+How Delloite uses KILT?
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:55]:
```suggestion
How does Delloite use KILT?
```

File: info-bites/july-2023.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+- Deloitte Consulting AG announced the integration of KILT blockchain technology for issuing reusable digital credentials to support KYC and KYB processes. These credentials have multiple use cases including regulatory compliance for banking and Decentralized Finance (DeFi), age verification for e-commerce, private logins and fundraising.
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:57]:
```suggestion
- Deloitte Consulting AG announced the integration of KILT blockchain technology for issuing reusable digital credentials to support  Know Your Client (KYC) and Know Your Business (KYB) processes. These credentials have multiple use cases, including regulatory compliance for banking and Decentralized Finance (DeFi), age verification for e-commerce, private logins and fundraising.
```

**Review by ChrisChinchilla — CHANGES_REQUESTED** [review]
@aybarsayan Few small tweaks

File: info-bites/july-2023.md (hunk 1 above)
Comment (aybarsayan) [reply info-bites/july-2023.md:1]: Yes, this ones are already used. I imported them to be a placeholder 😊

File: info-bites/july-2023.md (hunk 4 above)
Comment (aybarsayan) [reply info-bites/july-2023.md:39]: I was actually created for Ctypes too in earlier Info Bites

File: info-bites/july-2023.md (hunk 5 above)
Comment (aybarsayan) [reply info-bites/july-2023.md:49]: Yes I was 😊

Comment (ChrisChinchilla) [reply info-bites/july-2023.md:49]: OK @aybarsayan then…

```suggestion
🧐 Who is Deloitte? 🧐
```

File: info-bites/july-2023.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+So, with KILT DIDs, get ready to unlock your identity and embark on exciting journeys! 🗝️ Claim Independence! 🙌✨
+
+## 3
+
+![light vs full](files/deloitte.jpeg)
+
+🧐 Who is? 🧐
+
+Who is Delloite?
```
Comment (ChrisChinchilla) [inline info-bites/july-2023.md:51]:
```suggestion
```

**Review by ggera — APPROVED** [review]
I would make two subdirecotries `img` and `md` so that the root directory won't be floded with new files

**Pull request merged**